### ISSUE TITLE
Add discarding of orphan roos

### ISSUE THREAD

**Issue by kittsville** [kittsville/Pyroon#1]
Some roos imported just don't lead anywhere and become oprhan roos:
![Picture of orphan roo](https://cloud.githubusercontent.com/assets/6013087/21297469/04394132-c579-11e6-880a-9302ff1621f7.png)
These aren't very useful in the Cytoscape graph (web front-end) and can probably be discarded. They might be useful to be kept on a regular graph saving as they might become linked to by a roo added later. Could always make an option on export.

**tl;dr remove orphan roos from graph exported for web front-end**

**Comment by kittsville** [comment]
I might just leave it as is, with orphan roo removal working in-place.
